samples/net/azure_iot_hub/README.rst

```diff
@@ -89,12 +89,8 @@ Check and configure the following library options that are used by the sample:
 * :kconfig:option:`CONFIG_AZURE_IOT_HUB_DEVICE_ID` - Sets the Azure IoT Hub device ID. Alternatively, the device ID can be provided at run time.
 * :kconfig:option:`CONFIG_AZURE_IOT_HUB_HOSTNAME` - Sets the Azure IoT Hub host name. If DPS is used, the sample assumes that the IoT hub host name is unknown, and the configuration is ignored. The configuration can also be omitted and the hostname provided at run time.
 
-If DPS is used, use the Kconfig fragment found in the :file:`overlay-dps.conf` file and change the desired configurations there.
-As an example, the following compiles with DPS for the nRF9160 DK:
-
-.. code-block:: console
-
-	west build -p -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG=overlay-dps.conf
+If DPS is used, set the :makevar:`EXTRA_CONF_FILE` option to the :file:`overlay-dps.conf` file and change the desired configurations there.
```

Comment:
Here it's only _one_ relevant overlay file. I'd prefer just showing the command like we did before instead of only half explaining it and forcing the user to click a link to find out the rest.